apps/home/stylesheets/howitworks.styl

```diff
@@ -0,0 +1,102 @@
+.HomeHowItWorks
+  border-top 1px solid colors-gray-border
+  border-bottom 1px solid colors-gray-border
+
+  &.HomeSection__content
+    padding-bottom 4em
+    padding-left 0
+    padding-right 0
+
+  &__carousel
+    height 700px
+    width 100%
+
+.slick-slider 
+  position relative
+  display block
+  box-sizing border-box
+  user-select none
+  touch-action pan-y
+  -webkit-tap-highlight-color transparent
+  width 100%
+
+.slick-list 
+  position relative
+  overflow hidden
+  display block
+  margin 0
+  padding 0
+
+  &focus
+    outline none
+  
+  &.dragging 
+    cursor pointer
+    cursor hand
+  
+
+.slick-slider .slick-track,
+.slick-slider .slick-list 
+  transform translate3d(0, 0, 0)
+
+
+.slick-track 
+  position relative
+  left 0
+  top 0
+  display block
+  margin-left auto
+  margin-right auto
+  height 700px
+
+  &before,
+  &after 
+    content ""
+    display table
+  
+
+  &after 
+    clear both
+  
+
+  .slick-loading & 
+    visibility hidden
+  
+
+.slick-slide 
+  float left
+  height 100%
+  min-height 1px
+
+  [dir="rtl"] &
+    float right
+  
+  img 
+    display block
+  
+  &.slick-loading img 
+    display none
+  
+
+  display none
+
+  &.dragging img 
+    pointer-events none
+  
+
+  .slick-initialized & 
+    display block
+  
+
+  .slick-loading & 
+    visibility hidden
+  
+
+  .slick-vertical & 
+    display block
+    height auto
+    border 1px solid transparent
+  
+
+.slick-arrow 
+    display none
```

Comment:
Is this the only way to style this?

At a minimum can this be an isolated file with consistent formatting?

Reply:
Yes, it shall be done.